build/Build.cs

```diff
@@ -0,0 +1,494 @@
+
+            DockerTasks.DockerRun(_ => _
+                .EnableRm()
+                .EnableTty()
+                .SetEnv($"VERSION={OctoVersionInfo.FullSemVer}",
+                    $"BINARIES_PATH=/artifacts/OctopusTools.{OctoVersionInfo.FullSemVer}.linux-x64.extracted/",
+                    "PACKAGES_PATH=/artifacts",
+                    "SIGN_PRIVATE_KEY",
+                    "SIGN_PASSPHRASE")
+                .SetVolume(AssetDirectory + ":/BuildAssets",
+                    LinuxPackageFeedsDir + ":/opt/linux-package-feeds",
+                    ArtifactsDirectory + ":/artifacts")
+                .SetImage("octopusdeploy/package-linux-docker:latest")
+                .SetCommand("bash")
+                .SetArgs("/BuildAssets/create-octopuscli-linux-packages.sh"));
+
+            DeleteDirectory(ArtifactsDirectory / $"OctopusTools.{OctoVersionInfo.FullSemVer}.linux-x64.extracted");
+
+            var linuxPackagesDir = ArtifactsDirectory / $"linuxpackages";
+            EnsureExistingDirectory(linuxPackagesDir);
+            ArtifactsDirectory.GlobFiles($"*.deb").ForEach(path => MoveFile(path, linuxPackagesDir / new FileInfo(path).Name));
+            ArtifactsDirectory.GlobFiles($"*.rpm").ForEach(path => MoveFile(path, linuxPackagesDir / new FileInfo(path).Name));
+            CopyFileToDirectory($"{LinuxPackageFeedsDir}/publish-apt.sh", linuxPackagesDir);
+            CopyFileToDirectory($"{LinuxPackageFeedsDir}/publish-rpm.sh", linuxPackagesDir);
+            CopyFileToDirectory(AssetDirectory / $"repos/test-linux-package-from-feed-in-dists.sh", linuxPackagesDir);
+            CopyFileToDirectory(AssetDirectory / $"repos/test-linux-package-from-feed.sh", linuxPackagesDir);
+            CopyFileToDirectory($"{LinuxPackageFeedsDir}/test-env-docker-images.conf", linuxPackagesDir);
+            CopyFileToDirectory($"{LinuxPackageFeedsDir}/install-linux-feed-package.sh", linuxPackagesDir);
+            CompressionTasks.CompressZip(linuxPackagesDir, ArtifactsDirectory / $"OctopusTools.Packages.linux-x64.{OctoVersionInfo.FullSemVer}.zip");
+            TeamCity.Instance.PublishArtifacts(ArtifactsDirectory / $"OctopusTools.Packages.linux-x64.{OctoVersionInfo.FullSemVer}.zip");
+            DeleteDirectory(linuxPackagesDir);
+        });
+
+    [PublicAPI]
+    Target CreateDockerContainerAndLinuxPackages => _ => _
+        .DependsOn(BuildDockerImage)
+        .DependsOn(CreateLinuxPackages);
+
+    void SignBinaries(string path)
+    {
+        Logger.Info($"Signing binaries in {path}");
+        var files = Directory.EnumerateFiles(path, "Octopus.*.dll", SearchOption.AllDirectories).ToList();
+        files.AddRange(Directory.EnumerateFiles(path, "octo.dll", SearchOption.AllDirectories));
+        files.AddRange(Directory.EnumerateFiles(path, "octo.exe", SearchOption.AllDirectories));
+        //files.AddRange(Directory.EnumerateFiles(path, "octo", SearchOption.AllDirectories));
```

Comment:
This wasn't actually being signed in the cake script - confirm by checking the build logs